godtools/App/Features/Dashboard/Presentation/Tools/Subviews/ToolCard/ToolCardView.swift

```diff
@@ -153,11 +153,13 @@ struct ToolCardView: View {
                                 
                                 toolDetailsTappedClosure?()
                             }
+                            .buttonStyle(.plain)
                             
                             GTBlueButton(title: viewModel.openButtonTitle, font: navButtonFont, width: navButtonWidth, height: navButtonHeight, titleHorizontalPadding: navButtonTitleHorizontalPadding, accessibility: .openTool) {
-
+                                
                                 openToolTappedClosure?()
                             }
+                            .buttonStyle(.plain)
```

Comment:
How did this go?  Did you notice that all the buttons were firing when changing to List?

Reply:
Yes!  Thanks for the heads up on this!  I thought I had broken something at first before I remembered what you said.  Weird that it defaults to that behavior.  I ended up moving the .buttonStyle to apply to the entire view rather than each subview.  Which in theory should have removed the row highlighting, according to https://stackoverflow.com/questions/56545444/how-to-remove-highlight-on-tap-of-list-with-swiftui.  Wondering if iOS 18 has a bug because it seems like no one can get the highlight to remove consistently.  I've gone around and around with it and can't get it to go away 😩

Reply:
Ah says may apply a visual effect. (https://developer.apple.com/documentation/swiftui/plainbuttonstyle)